mini-bank/create-account.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
This is the landing page when the user has no account. The file name should reflect its function. So rename to `landing.html`